packages/extract-client-components/extract_client_components.ml

```diff
@@ -88,33 +75,25 @@ let capture_all_client_component_files_in_target path =
   in
   traverse_fs path
 
-open Cmdliner
-
-(* Command-line arguments *)
 let melange_target =
   let doc = "Path to the melange target directory (melange.emit (target xxx))" in
-  Arg.(required & pos 0 (some string) None & info [] ~docv:"MELANGE_TARGET" ~doc)
-
-let output_path =
-  let doc = "Output path for the generated manifest (defaults to './bootstrap.js')" in
-  Arg.(value & opt string "./bootstrap.js" & info [ "o"; "out" ] ~docv:"PATH" ~doc)
+  Cmdliner.Arg.(required & pos 0 (some string) None & info [] ~docv:"MELANGE_TARGET" ~doc)
 
-let extract_components target path =
+let extract_components target =
   let current_dir = Sys.getcwd () in
   let melange_target = Filename.concat current_dir target in
   match capture_all_client_component_files_in_target melange_target with
   | Ok manifest ->
-      render_manifest ~path manifest;
+      print_endline (render_manifest manifest);
```

Comment:
Why do we need to print it?

Reply:
we print to the stdout so I can use the CLI from both:

- dune `(with-stdout-to (cli))`
- esbuild `const content = execSync("cli"); Fs.writeFile(...)`

Before the cli was writing the file, so from the outside had less control. That could be configured by a parameter, but the stdout should probably be the default